Fastfile

```diff
@@ -140,10 +134,33 @@ def _build_with_gym()
   end
 end
 
+def _get_export_method()
+  ENV['GYM_EXPORT_OPTIONS'].nil? ? _fallback_to_enterprise_export_method() : get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: "method")
```

Comment:
Usually you'd omit the `()` if the method takes no params. Same on method declaration. But given we do `()` everywhere else in the file it seems fine.

Reply:
@neilkimmett Yeah that's what I thought as well. Could be a nice PR to tidy that up separately though